atom/browser/ui/message_box_gtk.cc

```diff
@@ -154,13 +169,16 @@ class GtkMessageBox : public NativeWindowObserver {
   }
 
   CHROMEGTK_CALLBACK_1(GtkMessageBox, void, OnResponseDialog, int);
+  CHROMEGTK_CALLBACK_0(GtkMessageBox, void, OnCheckboxToggled);
```

Comment:
@poiru I tested this out on Ubuntu 16.04 and initially saw this crash when `CHROMEGTK_CALLBACK_1` was being used:

```
Received signal 11 SEGV_ACCERR 000000cf5b00
#0 0x7fd99563fd07 <unknown>
#1 0x7fd98c02e390 <unknown>
#2 0x000000cf5d90 atom::(anonymous namespace)::GtkMessageBox::OnCheckboxToggled()
#3 0x000000cf5b20 atom::(anonymous namespace)::GtkMessageBox::OnCheckboxToggledThunk()
#4 0x7fd98a387fa5 g_closure_invoke
#5 0x7fd98a399fc1 <unknown>
#6 0x7fd98a3a2d5c g_signal_emit_valist
#7 0x7fd98a3a308f g_signal_emit
#8 0x7fd98bbc7553 <unknown>
#9 0x7fd98a3881d4 <unknown>
#10 0x7fd98a3a29a6 g_signal_emit_valist
#11 0x7fd98a3a308f g_signal_emit
#12 0x000000cf4cd4 atom::(anonymous namespace)::GtkMessageBox::GtkMessageBox()
```

Switching it to a `CHROMEGTK_CALLBACK_0` instead of `CHROMEGTK_CALLBACK_1` made it go away and provided the right checkbox value. It looked like the `gpointer` was unused so it seemed safe to ignore.